code/game/atoms_movable.dm

```diff
@@ -116,6 +120,34 @@
 		return 1
 	return 0
 
+/mob/living/forceMove(atom/destination)
+	stop_pulling()
+	if(pulledby)
+		pulledby.stop_pulling()
+	if(buckled)
+		buckled.unbuckle_mob()
+	if(buckled_mob)
+		unbuckle_mob(force=1)
+	. = ..()
+	if(client)
+		reset_perspective(destination)
+	if(pipes_shown.len && !istype(loc, /obj/machinery/atmospherics))
+		remove_ventcrawl()
+
+/mob/living/carbon/brain/forceMove(atom/destination)
+	if(container)
+		container.forceMove(destination)
+	else //something went very wrong.
+		CRASH("Brainmob without container.")
+
+
+/mob/living/silicon/pai/forceMove(atom/destination)
+	if(card)
+		card.forceMove(destination)
+	else //something went very wrong.
+		CRASH("pAI without card")

```

Comment:
Replacement for on_forceMove(), strangely on_forceMove() was not called by forceMove() but by admin_forceMove() only (which means there might have been some issue if a pai or brain called forceMove() ).